src/deadline/client/ui/widgets/host_requirements_tab.py

```diff
@@ -527,6 +612,28 @@ def _build_ui(self):
         # LineEdit  / LineEdit / Optional [X]
         self.layout.addLayout(self.columns)
 
+    def name(self) -> str:
```

Comment:
Nice to have: could we use `@property` and `@setter` methods here for all these properties ([ref](https://docs.python.org/3/library/functions.html#property))? Then we can just do
`amount.name = "amount.custom.test"`, and retrieve the value via `amount.name`. Similar note for the min and max values

Reply:
Good idea, working on that now